### PR TITLE
[Web Gradients] -> [Colors]

### PR DESCRIPTION
**Pull request by carlson-matt**
[Web Gradients] -> [Colors]
A cool place to find css gradients

Link: https://webgradients.com/

### CODE REVIEW

**Comment by Alaev** [comment]
Hey, @carlson-matt Can you change please the Pull request title to the following format?
[Web Gradients] -> [Colors]

And add the link to the pull request section?
Link: https://webgradients.com/

Great resource 👏

File: readme.md [hunk 1]
```diff
@@ -108,7 +108,8 @@
 | [Colorsinspo](https://colorsinspo.com/) | All-in-one resource for finding everything about colors |
 | [Happyhues](https://www.happyhues.co/) | Happy Hues is a color palette inspiration site that acts as a real world example as to how the colors could be used in your design projects by Mackenzie Child |
 | [Adobe Trends](https://color.adobe.com/trends)| Discover current color trends in different industries from the creative communities on Behance and Adobe Stock|
-| [Color Hunt](https://colorhunt.co/)| A free and open platform for color inspiration with thousands of trendy hand-picked color palettes |
+| [Color Hunt](https://colorhunt.co/)| A free and open platform for color inspiration with thousands of trendy hand-picked color palettes | 
+| [Web Gradients](https://webgradients.com/)| A free website to find good css gradients
```
Comment (Alaev) [inline readme.md:112]:
```suggestion
| [Web Gradients](https://webgradients.com/)| A free website to find good css gradients |
```

**Comment by carlson-matt** [comment]
> Hey, @carlson-matt Can you change please the Pull request title to the following format?
> [Web Gradients] -> [Colors]
> 
> And add the link to the pull request section?
> Link: https://webgradients.com/
> 
> Great resource 👏

Fixed!

**Comment by Alaev** [comment]
Title of the pr is Webgradients :)

**Comment by carlson-matt** [comment]
> Title of the pr is Webgradients :)

I see, i was editing the comment not the title

File: readme.md [hunk 2]
```diff
@@ -108,8 +108,9 @@
 | [Colorsinspo](https://colorsinspo.com/) | All-in-one resource for finding everything about colors |
 | [Happyhues](https://www.happyhues.co/) | Happy Hues is a color palette inspiration site that acts as a real world example as to how the colors could be used in your design projects by Mackenzie Child |
 | [Adobe Trends](https://color.adobe.com/trends)| Discover current color trends in different industries from the creative communities on Behance and Adobe Stock|
-| [Color Hunt](https://colorhunt.co/)| A free and open platform for color inspiration with thousands of trendy hand-picked color palettes |
-
+| [Color Hunt](https://colorhunt.co/)| A free and open platform for color inspiration with thousands of trendy hand-picked color palettes | 
+| [Web Gradients](https://webgradients.com/)| A free website to find good css gradients |
+| [Colorhexia](https://www.colorhexa.com/index.php) | A free collor lookup / pallet lookup tool |
```
Comment (Alaev) [inline readme.md:113]: can you add this one in a new Pull request?

**Pull request merged**